content/fr/blog/posts/mesurer-le-progrès :-un-modèle-de-maturité-pour-les-produits-du-gouvernement-numérique..md

```diff
@@ -1,12 +1,10 @@
 ---
-aliases:
-  - /2024/11/04/mesurer-le-progrès-un-modèle-de-maturité-pour-les-produits-du-gouvernement-numérique.
```

Comment:
These 2 lines need to be kept in the file in order for the redirect to work and maybe add a new line for the November 5th link
```
/2024/11/05/mesurer-le-progrès-un-modèle-de-maturité-pour-les-produits-du-gouvernement-numérique.
```